src/contracts/atlas/Atlas.sol

```diff
@@ -222,6 +223,9 @@ contract Atlas is Escrow, Factory {
             }
         }
 
+        // Write off the gas cost involved in on-chain bid-finding execution of all solverOps
+        _writeOffBidFindGasCost(_gasWaterMark - gasleft());
+
         // Reinitialize _bidsAndIndicesLastIndex to iterate through the sorted array in descending order
```

Comment:
It won't make a huge difference, but WDYT about moving this down so it also covers the `insertionSort()` costs?

```diff
- _writeOffBidFindGasCost(_gasWaterMark - gasleft());

  // Reinitialize _bidsAndIndicesLastIndex to iterate through the sorted array in descending order
  _bidsAndIndicesLastIndex = solverOpsLength - 1;

  // Then, sorts the uint256 array in-place, in ascending order.
  LibSort.insertionSort(_bidsAndIndices);

  ctx.bidFind = false;

+ _writeOffBidFindGasCost(_gasWaterMark - gasleft());
```

Reply:
yeah makes sense

Reply:
Done here https://github.com/FastLane-Labs/atlas/pull/371/commits/8c90a60eb07cf507b6d6890f7c6e823cc52abc9c